example/zipkin/main.go

```diff
@@ -58,7 +58,15 @@ func initTracer() {
 }
 
 func main() {
-	initTracer()
+	var url string
+
+	if len(os.Args) > 1 {
```

Comment:
Same, with regard to the `flag` package?

Reply:
@MrAlias i added git commit cd7d79c to resolve this change request